src/applications/check-in/hooks/useGetCheckInData.jsx

```diff
@@ -165,15 +164,12 @@ const useGetCheckInData = ({
           const { payload } = json;
           setPreCheckInData(payload);
 
-          if (payload.appointments?.length > 0) {
-            if (appointmentStartTimePast15(payload.appointments)) {
-              updateError('pre-check-in-past-appointment');
```

Comment:
We can delete a unit test and a case from `check-in/pre-check-in/pages/Error/index.jsx` that references this `pre-check-in-past-appointment`